site/content@beta/backend-apis/about-apis.md

```diff
@@ -0,0 +1,435 @@
+  "custom_id": "12312.301231238591134",
+  "order": 1,
+  "type": "annotation",
+  "author_id": "b7eff798-f8df-4364-8059-649c35c9ed0c",
+  "timestamp": 1473433500,
+  "text": "Sample annotation",
+  "annotation_type": "rating",
+  "properties": {
+    // "Properties" object
+  }
+}
+```
+This event type adds an annotation to the last thread. Sending an annotation **cannot** start a new thread, even when there's no active thread in a chat. It always goes to the latest already existing thread.
+
+The `annotation_type` field defines the type of annotation. It **cannot be empty**. It's customizable, so you can have your own annotation types. We used the `rating` type in the example.
+
+
+
+### Filled form
+
+A **filled form** is an event containing data from a form, such as the one in the picture below.
+
+
+> A sample **filled form**:
+
+```js
+{
+  "id": "0affb00a-82d6-4e07-ae61-56ba5c36f743",
+  "custom_id": "12312.301231238591134",
+  "order": 4,
+  "type": "filled_form",
+  "author_id": "b7eff798-f8df-4364-8059-649c35c9ed0c",
+  "timestamp": 1473433500,
+  "properties": {
+    // "Properties" object
+  }
+  "fields": [{
+      "type": "text",
+      "name": "name",
+      "label": "Your name:",
+      "required": true,
+      "value": "John Doe"
+    },
+    {
+      "type": "email",
+      "name": "email",
+      "label": "E-mail:",
+      "required": true,
+      "value": "john.doe@gmail.com"
+    },
+    {
+      "name": "Chat window title",
+      "type": "title",
+      "label": "Let's talk!",
+    },
+    {
+      "name": "Chat window form info",
+      "type": "information",
+      "label": "Before we start, we'd like to know a few details about you.",
+    }]
+}
+```
+![Filled Form](filled_form.png "filled form example")
+
+To send the data from this form, you can use the **filled form** event. The code sample introduces all currently available field types:
+
+- `type`
+- `name`
+- `label`
+- `required`
+- `value`
+
+### System message
+
+> A sample **system message**:
+
+```js
+{
+  "id": "0affb00a-82d6-4e07-ae61-56ba5c36f743",
+  "order": 1,
+  "type": "system_message",
+  "timestamp": 1473433500,
+  "text": "Mike joined the chat",
+  "system_message_type": "agent_joined"
+}
+```
+A **system message** is an event generated by a server. It doesn't contain the `author_id`, `custom_id`, or the `properties` field.
+
+System messages can be triggered by:
+
+- a user action within a chat 
+- a router (router messages are covered in the [Router system messages](#router-system-messages) section) 
+
+Each action triggers a system message with a different `system_message_type` and a different `text` message.
+
+| system_message_type | text                             |
+| ------------------- | -------------------------------- |
+| `agent_joined`      | `<agent_name> joined the chat`   |
+| `agent_left`        | `<agent_name> left the chat`     |
+| `manual_archived`   | `<agent_name> archived the chat` |
```

Comment:
typów jest dużo więcej to chyba tylko przykłady?

Reply:
table was removed